clang/lib/Parse/ParseDecl.cpp

```diff
@@ -8119,6 +8119,14 @@ void Parser::ParseParameterDeclarationClause(
     }
 
     if (TryConsumeToken(tok::ellipsis, EllipsisLoc)) {
+      if (getLangOpts().CPlusPlus26) {
+        // C++26 [dcl.dcl.fct]p3:
+        //   A parameter-declaration-clause of the form
+        //   parameter-list '...' is deprecated
+        Diag(EllipsisLoc, diag::warn_deprecated_missing_comma_before_ellipsis)
+            << FixItHint::CreateInsertion(EllipsisLoc, ", ");
+      }
+
```

Comment:
below:  `else if (!getLangOpts().CPlusPlus) {`

Reply:
I had intended to fallthrough to those if blocks so that the `else if (ParmDeclarator.getEllipsisLoc().isValid() || Actions.containsUnexpandedParameterPacks(ParmDeclarator))` block would be evaluated to also emit the parameter pack diagnostics. Do you think those should be omitted in this case?

Reply:
Hum, nah, lets leave it like that